.gitignore

```diff
@@ -63,6 +63,7 @@ node_modules/
 
 # react webapp gitignore
 webapp/node_modules
+webapp/package-lock.json
```

Comment:
This should not be ignored, applications should have pinned deps.

Reply:
Thanks for pointing that out.